jetty-core/jetty-http/src/main/java/org/eclipse/jetty/http/MutableHttpFields.java

```diff
@@ -495,8 +495,8 @@ public int nextIndex()
         @Override
         public HttpField previous()
         {
-            if (_index == 0)
-                throw new NoSuchElementException("-1");
+            if (_index <= 0)
+                throw new NoSuchElementException(Integer.toString(_index - 1));
```

Comment:
If inequality here, let's use inequality also for `next()`.
In this way we are also consistent with `ImmutableHttpFields` where inequality is used.